components/mpas-ocean/src/shared/mpas_ocn_tracer_ecosys.F

```diff
@@ -1013,7 +1013,10 @@ subroutine ocn_tracer_ecosys_compute(activeTracers, ecosysTracers, forcingPool,
          call marbl_instances(1)%interior_tendency_compute()
 
          if (marbl_instances(1)%StatusLog%labort_marbl) &
-           call mpas_log_write("WARNING: problem in MARBL interior_tendency_compute", MPAS_LOG_CRIT)
+!     maltrud ignore abort
+!          call mpas_log_write("WARNING: problem in MARBL interior_tendency_compute", MPAS_LOG_CRIT)
+! write(*,*)' IGNORING problem in MARBL interior_tendency_compute'
```

Comment:
Is this critical log write not actually a problem? If not, it should just be removed.